bin/npg_check_study_staging_location

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use FindBin qw($Bin);
+use lib ( -d "$Bin/../lib/perl5" ? "$Bin/../lib/perl5" : "$Bin/../lib" );
+use Log::Log4perl qw(:levels);
+use Getopt::Long;
+use Pod::Usage;
+use Readonly;
+use Try::Tiny;
+use Carp;
+use English qw(-no_match_vars);
+use MIME::Lite;
+
+use WTSI::DNAP::Warehouse::Schema;
+use npg_tracking::Schema;
+
+Readonly::Scalar my $EMAIL_DOMAIN => 'sanger.ac.uk';
+Readonly::Scalar my $FOLDER_PATH => 3;
+
+our $VERSION = '0';
+
+my $help;
+my $verbose = 0;
+my $study = q(UK Biobank Main Phase Study);
+my $staging_areas = q();
+my $emails = q();
+my $num_days = 1;
+
+GetOptions (
+            'help'         => \$help,
+            'verbose'      => \$verbose,
+            'study=s'      => \$study,
+            'num-days=i'   => \$num_days,
+            'staging-areas=s' => \$staging_areas,
+            'emails=s'        => \$emails,
+           );
+
+if ($help) { pod2usage(0); }
+
+my @staging_array = split /,/smx, $staging_areas;
+my @email_array = split /,/smx, $emails;
+@email_array = map { /@/sxm ? $_ : $_.q(@).$EMAIL_DOMAIN } @email_array;
+
+my $layout = '%d %-5p %c - %m%n';
+Log::Log4perl->easy_init({layout => $layout,
+                          level  => $INFO,
+                          utf8   => 1});
+my $logger = Log::Log4perl->get_logger();
+
+my $warehouse_schema = WTSI::DNAP::Warehouse::Schema->connect()->storage->dbh();
+my $tracking_schema = npg_tracking::Schema->connect()->storage->dbh();
+
+# run-in-progress is status = 2
+## no critic (ValuesAndExpressions::ProhibitImplicitNewlines)
+my $SQL_runs_in_progress = q{
+select distinct r.id_run,
+r.batch_id,
+rs.date,
+r.folder_path_glob
+from run r,
+run_status rs
+where r.id_run=rs.id_run
+and rs.iscurrent=1
```

Comment:
```suggestion
```
Do we only care that "run in progress" was _started_ within `$num_days` and not that it is currently "run in progress"?